locale/pl/LC_MESSAGES/amo.po

```diff
@@ -78,20 +76,19 @@ msgstr "Używane przez"
 
 #: src/amo/components/AddonMoreInfo.js:24
 msgid "Homepage"
-msgstr ""
+msgstr "Strona domowa"
 
 #: src/amo/components/AddonMoreInfo.js:32
 msgid "Support Site"
-msgstr ""
+msgstr "Strona pomocy"
```

Comment:
I don't know if I get support as a place idea right but I think we should make it clear in https://github.com/aviarypl/team/issues/4 (and fix here later if needed)